### DISCUSSION TITLE
Welcome to ZipChord Discussions!

### DISCUSSION THREAD

**Discussion by psoukie** [psoukie/zipchord#20] [Announcements]
## 👋 Welcome!
This is as a place to provide feedback and discuss ZipChord, and typing, chording and stenotyping in general. Please use Discussions to:
  * Ask questions about ZipChord, typing, chords, dictionaries, keyboards etc.
  * Share feedback and ideas about ZipChord.
  * Engage with other people focused on typing efficiencies and innovations.

To get started, comment below with feedback, question, or an introduction of yourself. Feel free to also share how you use (or want to use) the combination of chords and typing in ZipChord.

**Comment by dzeruel** [comment]
This is amazing psoukie!! I was about to write this myself and low and behold I've found your program! Amazing thank you so much! I am planning to write a dictionary for it with a help of a script which turns a given word into the optimal keychord.

**Reply by psoukie** [reply]
Thank you! Ping me with any suggestions or ideas you come across as you use it. And thanks for the bug report about the special character.
